Login page and recover page Cypress tests

JimmyOei opened the pull request:
**Description**
This PR consists of the first Cypress tests for the front-end testing, namely tests for the login page and the recover page. Also including some tools that helps with commonly used implementations and site navigations for all the Cypress tests.

_Changes in detail_
- **Adds** a directory for the login page tests, consisting of tests for all the buttons and fields on this page and the modal alert popup when trying logging in.
- **Adds** a directory for the recover page tests, consisting of tests for the button and field on this page and the modal alert popup when trying to send the recovery link.
- **Changes** to the cypress config file for the environment variables of pages and the base url.

**Fixes _issue or discussion number_**

Works on the issue #3531.

Felienne commented:
ok I managed to run the tests on the command line: 

<img width="993" alt="image" src="https://user-images.githubusercontent.com/1003685/199096043-b65865f8-c483-4019-b39a-e7bccd4b9684.png">

Some are failing though and I am not sure that is expected? If you add instructions on how we should run the tests so we can verify it works as proposed that would be great!

JimmyOei commented:
I removed the two tests that didn't work and will further work on this in a new branch. The problem is that it involves loggin in a user that should be registered, but the user registered locally on one of our laptops is, ofcourse, not registered in the official hedy database or any other machine you are running the test on. Current tests should work and I added a description on how to run the tests.
